<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>411</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>상품명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>가격</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>종류</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>사진</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>제품 추가하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 24pt &quot;배달의민족 도현&quot;;

</string>
    </property>
    <property name="text">
     <string>StarBucks</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;굴림&quot;;</string>
    </property>
    <property name="text">
     <string>manager_mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_remove">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>710</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>제품 삭제하기</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
